PickUpEventsApp.Data/Model/Event.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace PickUpEventsApp.Data.Model
+{
+    public class Event
+    {
+        public User Owner { get; set; }
+
+        public string Name { get; set; }
+
+        public string Description { get; set; }
+
+        public IEnumerable<User> Users { get; set; } 
+
+        // This is a circular reference to this class.
+        public IEnumerable<Event> Events { get; set; }
```

Comment:
I am curious on what the circular reference means?

To make sure I understand properly, will Events have a collection of Events that belong to them?

Reply:
Discussed in meeting, property removed.